views/assets/styles/components/_pagination.scss

```diff
@@ -0,0 +1,59 @@
+$module: 'pager';
```

Comment:
This might be shared with UI.
If that is the case then UI-Elements is a great candidate for a shared module/component.